vnext/Microsoft.ReactNative/IReactContext.idl

```diff
@@ -90,6 +90,14 @@ namespace Microsoft.ReactNative
       "The name of the JavaScript bundle file to load. This should be a relative path from @.BundleRootPath. "
       "The `.bundle` extension will be appended to the end, when looking for the bundle file.")
     String JavaScriptBundleFile { get; };
+
+    DOC_STRING(
+        "A read-only snapshot of the @ReactInstanceSettings.JsiEngineOverride property value "
+        "at the time when the React instance was created.\n"
+        "")
+    UInt32 JsiEngine {
```

Comment:
this should use the type `JSIEngine` instead of `UInt32`

Reply:
(similar with the implementation, of course :) )

Reply:
I've some trouble in reusing the JSEngine enum which is defined in another IDL file .. If i redefine JSEngine enum in this file, the compiler complains about redefinition .. I could define a new enum and use it, but it looks confusing to define yet another enum for JS engines.. For now, there is no need to add this field to enum, hence removing the field for now.